index.js

```diff
@@ -1,22 +1,40 @@
 const { getAlerts } = require("./https/alerts");
 const { stage } = require("./https/stage");
 const { participate, participateStatus } = require("./https/participate");
-const { assignedGiftees, updateApiKey, volunteer } = require("./https/gw2Accounts")
-const { sendGift, receiveGift, reportGift } = require("./https/gifts")
-const { getNotSent, getNotReceived, getReported, getStats} = require("./https/stats")
+const {
+  assignedGiftees,
+  updateApiKey,
+  volunteer
+} = require("./https/gw2Accounts");
+const {
+  updateGiftSentStatus,
+  updateGiftReceivedStatus,
+  updateGiftReportedStatus
+} = require("./https/gifts");
+const {
+  getNotSent,
+  getNotReceived,
+  getReported,
+  getStats
+} = require("./https/stats");
```

Comment:
I am going to presume your IDE did this? Splitting it out onto a pile of newlines

Reply:
Yep! This is Prettier's doing. Imo it's easier to read.

Reply:
/me takes Prettier behind teh shed, a loud bang happens and Silver returns alone....